client/components/SurrenderForm.js

```diff
@@ -0,0 +1,188 @@
+import React, { useState } from "react"
+import ErrorList from "./ErrorsList"
+
+const SurrenderForm = (props) => {
+  const [message, setMessage] = useState("")
+  const [errors, setErrors] = useState({})
+  const [surrender, setSurrender] = useState({
+    personName: "",
+    phoneNumber: "",
+    email: "",
+    petName: "",
+    age: "",
+    petTypeId: "",
+    imgUrl: "",
+    vaccinationStatus: "",
+    adoptionStory: "",
+  })
+
+  const handleInputChange = (event) => {
+
+    setSurrender({
+      ...surrender,
+      [event.currentTarget.id]: event.currentTarget.value,
+    })
+  }
+
+  const submitHandler = (event) => {
+    event.preventDefault()
+    if (isValid()) {
+      addPetForSurrender()
+    } else {
+      console.log("form Not submitted!")
+    }
+  }
+
+  const addPetForSurrender = async () => {
+    try {
+      const response = await fetch("/api/v1/surrender", {

```

Comment:
good! to be a bit more RESTful/conventional, we could make this pluralized `surrenders`

Reply:
Thank you! Updated